bin/caret-create-folds.R

```diff
@@ -21,15 +21,15 @@ option_list = list(
     help = 'Number of groups to split the data in.'
   ), 
   make_option(
-    c("-o", "--output-cell-indexes"),
+    c("-o", "--output-dir"),
```

Comment:
If we just output a simple integer-named file to a directory, we can use that integer to add 'k' to the channels later on (see the .map{} added to main.nf).